### ISSUE TITLE
Navbar: nav items with menus don't indicate focus visually (2.4.7)

### ISSUE THREAD

**Issue by Blackbaud-MattGregg** [blackbaud/skyux1#874]
### Expected behavior

The nav item visually indicates it is focused when it receives focus. 
### Actual behavior

Nav items which have menus don't have visual indication. Note that nav items which don't have menus do have a visual indication when focused.
